analytics/analytics.go

```diff
@@ -28,6 +28,7 @@ func detectCI() (string, bool) {
 		"GITHUB_WORKFLOW": "github-actions",
 		"CIRCLECI":        "circle-ci",
 		"JENKINS_HOME":    "jenkins",
+		"AGENT_WORKDIR":   "jenkins", // https://github.com/jenkinsci/docker-agent/blob/master/11/alpine/Dockerfile#L35
```

Comment:
Can we move this to the bottom of the list, just in case it overlaps with a different CI system? I'm worried about this conflicting with a different CI system down the road.

Reply:
Yeah, good idea.